### ISSUE TITLE
Cant do any transactions because i put all of my FIS in attached Ledger to polkadot{.js} extension

### ISSUE THREAD

**Issue by aaasalimi** [stafiprotocol/stafi-node#7]
Dear Owner

I can see my FIS in attached Ledger to polkadot{.js} extension but can't do any transactions. 
I beg you fix this issue as soon as possible. 

Thanks in advance

![aa](https://user-images.githubusercontent.com/66446463/140987683-f2e05a29-1653-41e6-8678-1bf15004c022.jpg)

**Comment by Tore19** [comment]
Yeah, it doesn't support ledger for StaFi for now. We will consider supporting ledger in December. Sorry for the inconvenience.
